SCons/Conftest.py

```diff
@@ -231,13 +231,15 @@ def _check_empty_program(context, comp, text, language, use_shared: bool = False
         return context.CompileProg(text, suffix)
 
 
-def CheckFunc(context, function_name, header = None, language = None):
+def CheckFunc(context, function_name, header = None, language = None, funcargs = None):
```

Comment:
Both `CheckFunc` entries should have a change notice.
```
    .. versionchanged:: 4.7.0
       The ``funcargs`` parameter was added.
```

Reply:
Added in next commit.